guides/extensions/transport-ble/setup.md

```diff
@@ -0,0 +1,4 @@
+# Setup
```

Comment:
This doesn't really add anything. 

Maybe we can add it to the list in the extensions index and wait with adding this until we add some content?

Reply:
Yeah. I'm still adding the content. I'll request a review when I'm done

Reply:
Ah in that case it's better to set the PR as draft. That way reviewers know they don't have to look at the PR yet